src/components/channel/ChannelBadge.js

```diff
@@ -1,25 +1,29 @@
 import React from 'react'
 
-import { SubscribeButton } from './SubscribeButton'
+import SubscribeButton from './SubscribeButton'
 
-export const ChannelBadge = ({
+export default ({
   children,
   className = '',
   channel,
   hideInfo,
+  hideSubscribe,
   ...rest
 }) => (
   <div className={`channel-badge ${className}`} {...rest}>
     <div className='thumbnail-wrapper'>
-      <img className='thumbnail' src={channel.profileImage} />
+      <img className='thumbnail' src={channel.profileImage || 'http://placehold.it/60x60'} />
```

Comment:
Just go ahead and remove the placeholdit image—we can make a `ProfileImage` component later and have it choose a custom default.